### ISSUE TITLE
Title of modal "_manage_list" for toolbar button "link" has wrong title

### ISSUE THREAD

**Issue by philipptempel** [octobercms/october#1755]
**Bug summary:**
Title of the modal for toolbarButton "link" has wrong title

**Request location:**
n/a

**Reproduce steps:**
Given two models
    * Motionpattern hasMany Robots
    * Robot belongsTo Motionpattern
Controller "Robots" implements 'Backend.Behaviors.RelationController'

robots/fields.yaml:
```yaml
fields:
    title:
        label: Title
        type: text

    slug:
        preset: title
        label: Slug
        type: text

    motionpattern:
        label: Motionpattern
        type: partial
        path: field_motionpattern
```

config_relation.yaml:
```yaml
motionpattern:
    label: Motionpattern
    view:
        list:
            columns:
                title:
                    label: Title
                    searchable: true
                created_at:
                    label: Created at
                    type: timetense
        form:
            fields:
                title:
                    label: Title
                    type: text
        toolbarButtons: link|create
    manage:
        list: $/philipptempel/demo/models/motionpattern/columns.yaml
        form: $/philipptempel/demo/models/motionpattern/fields.yaml
        showSearch: true
        recordsPerPage: 10
```
Then clicking on the "Link Motionpattern" button creates a modal title "Add a new Motionpattern".

**Expected result:**
Title of modal should be matching the action i.e., should be "Link existing Motionpattern"

**Actual result:**
Clicking on the "Link Motionpattern" button (cf. first image) creates a modal title "Add a new Motionpattern" (cf. second image)

![screen shot 2016-02-05 at 11 35 21 am](https://cloud.githubusercontent.com/assets/1541337/12843933/a692a94a-cbfc-11e5-8fe8-78ed3e8d4d47.png)
![screen shot 2016-02-05 at 11 35 27 am](https://cloud.githubusercontent.com/assets/1541337/12843934/a6936330-cbfc-11e5-846b-c522e15bba25.png)

**Comment by daftspunk** [comment]
##### This bug has been fixed via live stream:
:movie_camera: https://youtu.be/ieNVvLwd5_w

**Comment by philipptempel** [comment]
Thanks @daftspunk. Watched your video. You mentioned something about that I might have gotten the relations mapping or name syntax incorrectly. Maybe I did, I'm still trying to figure out OctoberCms and Laravel within October. Coming from a FuelPHP background the way I took seemed logical to me. Maybe, even though it's a bit unrelated to this issue, could elaborate on my situation:

I have a model Motionpattern (or Post to be more general) which has many model Robots (or Comments). A Robot (or Comment) can only belong to one Motionpattern at a time but must belong to one as well (it's a required relation). I could as well implement this in such ways that I'll have an enum column on the Robot (Comment) and put the possible Motionpatterns (Posts) into the form config file. But I want to allow people to add a new Motionpattern (Post) if it doesn't exist yet. Either create the Motionpattern directly through the Motionpatterns Controller (Posts Controller) or through the Robots Controller (Comment Controller) when the specific Motionpattern does not exist on creating the Robot.
So it is similar to the setup you had in your video except for your case you cannot create a comment before having a post to "link" it to. In my case, I want to go the other way and allow adding a new Robot/Comment and defer creation of the Motionpattern/Post.
Makes sense?

PS: It's "Philipp Tempel" not "Philip P Tempel", but still cool you looked at my issue in a live stream :+1:
